luaui/Widgets/unit_key_select.lua

```diff
@@ -0,0 +1,29 @@
+---@diagnostic disable-next-line: duplicate-set-field
+function widget:GetInfo()
+	return {
+		name = "KeySelect",
+		desc = "Selects units on keypress.",
+		author = "woss (Tyson Buzza)",
+		date = "Aug 24, 2024",
+		license = "Public Domain",
+		layer = -999999,
```

Comment:
why -999999?

Reply:
No idea. I think I was copying the smart select widget. I don't know what that number does.

Reply:
You can replace it with 1, the layer is the loading order of the widget in relation to other widgets. We don't expect any conflicts to exist unless a game or user already declares another `select` action in a widget.